### PR TITLE
Handles delete operations

### PR DESCRIPTION
**Pull request by zcoursey22**
@kristie-zheng could you review this PR for me? Thanks

### CODE REVIEW

File: db/dbHelpers.js [hunk 1]
```diff
@@ -41,6 +41,14 @@ const addAppointment = (data, callback) => {
   });
 };
 
+const removeAppointment = (data, callback) => {
+  const queryString = `DELETE FROM appointments_log WHERE id=${data.appointment_id} AND business_id=${data.business_id}`;
```
Comment (kristiedesiree) [inline db/dbHelpers.js:45]: I don't know if this is necessary but there might be a constraint you can add here that assures that the record is only deleted if it exists (not sure how error handling works in SQL)

**Review by kristiedesiree — COMMENTED** [review]
Looks good, just had one comment/question there

**Review by kristiedesiree — APPROVED** [review]
This file looks good!

File: db/dbHelpers.js (hunk 1 above)
Comment (zcoursey22) [reply db/dbHelpers.js:45]: It looks like this shouldn't throw an error and simply nothing will be deleted if nothing matches the where clause. Thanks for bringing that to my attention though

**Pull request merged**